packages/integrations/cloudflare/README.md

```diff
@@ -191,6 +191,42 @@ export default defineConfig({
 });
 ```
 
+## WASM module imports
+
+`wasmModuleImports: boolean`
+
+Cloudflare has native support for importing `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). To enable importing them as modules in the cloudflare build and astro dev server, add `wasmModuleImports: true` to your config.
+
+```diff
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
++       wasmModuleImports: true
+    }),
+	  output: 'server'
+})
+```
+
+Once enabled, you can import a web assembly module in astro with a `.wasm?module` import. The integration supports WASM module imports in both server and hybrid mode for a consistent development experience.
```

Comment:
```suggestion
Once enabled, you can import a web assembly module in Astro with a `.wasm?module` import. 

The following example shows... (importing ... then ...)
```
Just a reminder that proper names, when we just mention them in a sentence, should be capitalized.

Ideally, we prefer to introduce code samples with a sentence like, "The following example shows... " and then an explanation of exactly what the code is doing. This means people don't have to both figure out what this example is, and then also how the thing you're demonstrating also works. We help them by telling them before they see the example that this is what the code accomplishes.

Also, the fact that it works in both modes is great to include! But, it seems out of place here. I suggest moving this sentence below the code sample, since it's not actually related to this code example.

Reply:
Good points. I made the capitalization of `Wasm` consistent with that suggested at [https://webassembly.org/](https://webassembly.org/). And updated example introduction text to be more descriptive